res/res_pjsip_endpoint_identifier_ip.c

```diff
@@ -241,6 +264,42 @@ static int header_identify_match_check(void *obj, void *arg, int flags)
 	return 0;
 }
 
+/*! \brief Comparator function for matching an object by request line */
+static int request_identify_match_check(void *obj, void *arg, int flags)
+{
+	struct ip_identify_match *identify = obj;
+	struct pjsip_rx_data *rdata = arg;
+
+	if (ast_strlen_zero(identify->match_request)) {
+		return 0;
+	}
+
+	int len;
+	char buf[PATH_MAX];
+
+	/* Print request line to req_buf */
+	len = pjsip_uri_print(PJSIP_URI_IN_REQ_URI, rdata->msg_info.msg->line.req.uri, buf, sizeof(buf) - 1);
```

Comment:
Is this really just printing out the request line URI? If so, that's not doing what the documentation states. The documentation states the request line which should include the SIP request method and the end.

Reply:
Indeed it prints the request URI. Without the SIP method in the beginning, or the protocol version in the end. Also, that was my intention anyway, so I renamed it everywhere.